jobs/periodic/sync-to-webflow/sync-courses.ts

```diff
@@ -0,0 +1,163 @@
+import { createNewItem, deleteItems, emptyMetadata, getCollectionItems, publishItems, WebflowMetadata } from './webflow-adapter';
+import { diff, hash, mapDBIdToId, DBIdMap } from './diff';
+import { Logger } from 'log4js';
+import moment, { Moment } from 'moment';
+import { accessURLForKey } from '../../../common/file-bucket';
+import { WebflowSubcourse, getWebflowSubcourses } from './queries';
+import { lectureDTOFactory } from './sync-lectures';
+
+// This is needed so that the weekday will be translated properly.
+moment.locale('de');
```

Comment:
Does this impact all uses of moment? If yes, can we move this into some higher level file?

Reply:
Yes and good point. We should not set a global value in any file.
I'll move the setting to locals.